kardinal/resources/resources.go

```diff
@@ -161,46 +162,56 @@ func AddAnnotations(obj *metav1.ObjectMeta, annotations map[string]string) {
 	}
 }
 
-// OPERATOR-TODO: Add create, update and delete global options
-// OPERATOR-TODO: Refactor the Apply... functions
-
-func ApplyServiceResources(ctx context.Context, clusterResources *Resources, clusterTopologyResources *Resources, cl client.Client) error {
+func ApplyResources(
+	ctx context.Context,
+	// Current cluster resources
+	clusterResources *Resources,
+	// Base + flows resources to reconcile
+	clusterTopologyResources *Resources,
+	cl client.Client,
+	// Function to retrieve the namespace resources
```

Comment:
What do you think of moving these comments to a docs comment format over the function header?
https://tip.golang.org/doc/comment#func